src/main/resources/static/js/results.js

```diff
@@ -0,0 +1,50 @@
+function exportToCSV() {
+
+    var url = "/export/";
+
+    //get the selected values to export CSV
+    var selectedVals = $("div.card-body input:checked").next('label').map(function () {
+        return $(this).text();
+    }).get();
+
+    //checking if any values were selected
+    if (selectedVals.length !== 0) {
+        var csvExportValsArr = [];
+
+        //populate the array as per the selected values
+        if (selectedVals.includes("Course")) {
+            csvExportValsArr.push("Course: ");
+            populateCsvVals(csvExportValsArr, "td:eq(0)");
+            csvExportValsArr.push("$");
+        }
+        if (selectedVals.includes("Category")) {
+            csvExportValsArr.push("Category: ");
+            populateCsvVals(csvExportValsArr, "td:eq(2)");
+            csvExportValsArr.push("$");
+        }
+        if (selectedVals.includes("Learning Outcomes")) {
+            csvExportValsArr.push("Learning Outcomes: ");
+            populateCsvVals(csvExportValsArr, "td:eq(3)");
+        }
+
+        var joinedVals = csvExportValsArr.join(" , ");
+        url += joinedVals;
+    } else {
+        url += " ";
+    }
+
+    //replaces the specific fragment
+    $("#exportCsv").load(encodeURI(url));
+
+}
+
+function populateCsvVals(csvExportValsArr, colNum) {
+    $("#courseResults tr").each(function () {
```

Comment:
I'm not seeing anything with the ID courseResults, the table's ID is results-table. Is it possible this is a mistake, because right now my CSV shows nothing: 

![image](https://user-images.githubusercontent.com/15951317/54608212-9de8c080-4a26-11e9-9fa7-e2aeb6572cdd.png)